### PR TITLE
Tenants improvements & new Development form

### PR DESCRIPTION
**Pull request by AleksandrSokolov**
1. _sc_tenants db changes & SC logic to support - new colunms:
 * created - timestamp when template was created, 
 * template - template that was used to create tenant 
2. populates _sc_tenants.email by email of user that creates this tenant
3. a lot of improvements in UI and logic for tenants - more information shown and etc
4. extends creation of tenant by add ability to write tenant description and add customising create template warning text (new SC parameter in config.ts and tenant config UI )
5. Adds database instance information into Settings / About application / System 
6. A lot of javascript warning are cleaned in few other places    
**** Also includes:
1. Add new screen "Development"
2. Move development setting into it
3. More i18 (internationalization)
**** Next plans (not implemented)
1. Improve Test coverage for tenants
2. A little bit of User management for tenants from main tenant admin UI (manage admin users)
3. block / unblock tenants

### CODE REVIEW

File: packages/server/routes/admin.js [hunk 1]
```diff
@@ -886,9 +875,25 @@ router.get(
                   ),
                   tr(th(req.__("Node.js version")), td(process.version)),
                   tr(
-                    th(req.__("Database")),
+                    th(req.__("Database type")),
                     td(db.isSQLite ? "SQLite " : "PostgreSQL ", dbversion)
                   ),
+                  tr(
```
Comment (glutamate) [inline packages/server/routes/admin.js:881]: I don't think it is a good idea to expose host, port and user (schema is fine). On some platforms (e.g. Heroku) these are secret and would expose this information to sub-tenant owners, allowing them to further probe the DB host.

**Review by glutamate — CHANGES_REQUESTED** [review]
Thanks, this is great, just a small comment

File: packages/server/routes/admin.js (hunk 1 above)
Comment (AleksandrSokolov) [reply packages/server/routes/admin.js:881]: Yes, better to show this only for main site for admin users only. I'll update this now

Comment (AleksandrSokolov) [reply packages/server/routes/admin.js:905]: Updated.

**Review by glutamate — APPROVED** [review]

**Pull request merged**